src/pybind/mgr/cephadm/module.py

```diff
@@ -2525,6 +2548,11 @@ def tuned_profile_rm_setting(self, profile_name: str, setting: str) -> str:
         self._kick_serve_loop()
         return f'Removed setting {setting} from tuned profile {profile_name}'
 
+    @handle_orch_error
+    def service_discovery_dump_cert(self) -> str:
+        root_cert = self.get_store('service_discovery/root/cert')
```

Comment:
This string appears another 2 times. What about a global constant?

```suggestion
        root_cert = self.get_store(KV_STORE_SD_ROOT_CERT)
```